File: GymGGun(그치만 남들 쉴 때…)/final_project_be/src/main/java/com/example/final_project_be/health/HealthController.java
package com.example.final_project_be.health;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.UUID;

// 서버가 정상적으로 작동하고 있는지 확인하기 위해 사용
@RestController
@RequestMapping("/health")
public class HealthController {

    private final String uniqueId = UUID.randomUUID().toString();

    @GetMapping
    public String checkHealth() {return "Server ID" + uniqueId;}
}

File: Bask-EAT((주)식회사)/Bask-EAT-Service-Backend/bin/main/com/bask_eat/business_backend/controller/HealthController.kt
package com.bask_eat.business_backend.controller

import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api")
class HealthController {

  @GetMapping("/health")
  fun health(): ResponseEntity<Map<String, String>> {
    return ResponseEntity.ok(mapOf("status" to "UP"))
  }
}


